dist/index.d.ts

```diff
@@ -39,17 +39,23 @@ declare module "shp-write" {
     arraybuffer: ArrayBuffer;
     blob: Blob;
     nodebuffer: Buffer;
-}
-type OutputType = keyof OutputByType;
+  }
+  type OutputType = keyof OutputByType;
 
   export interface ZipOptions {
     compression: Compression,
     type: OutputType
   }
 
+  DEFAULT_ZIP_OPTIONS = {
+    compression: 'STORE',
+    type: 'base64'
+  }
+
   export function download(
     geojson: GeoJSON.FeatureCollection,
-    options?: DownloadOptions
+    options?: DownloadOptions,
+    zipOptions?: ZipOptions = DEFAULT_ZIP_OPTIONS
```

Comment:
This is slightly confusing here. It may be better to eventually remove the download function, but for now I think having a single options parameter is better than having two.